dolphinscheduler-task-plugin/dolphinscheduler-task-tis/pom.xml

```diff
@@ -60,10 +65,22 @@
             </exclusions>
         </dependency>
 
+<!--        <dependency>-->
+<!--            <groupId>org.asynchttpclient</groupId>-->
+<!--            <artifactId>async-http-client</artifactId>-->
+<!--            <version>2.12.3</version>-->
+<!--        </dependency>-->
         <dependency>
-            <groupId>org.asynchttpclient</groupId>
-            <artifactId>async-http-client</artifactId>
-            <version>2.12.3</version>
+            <groupId>org.jacoco</groupId>
+            <artifactId>org.jacoco.agent</artifactId>
+            <classifier>runtime</classifier>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.java-websocket</groupId>
+            <artifactId>Java-WebSocket</artifactId>
+            <version>1.5.1</version>
```

Comment:
version need to be declared in the main pom file

Reply:
has register dependency Java-WebSocket in root pom